tests/protocols/guidance_test.py

```diff
@@ -0,0 +1,178 @@
+from client.protocols import unlock
+from client.protocols import guidance
+from client.sources.common import models
+import mock
+import unittest
+import os
+
+
+class GuidanceProtocolTest(unittest.TestCase):
+	GUIDANCE_DIRECTORY = "demo/guidance_test/"
```

Comment:
Why is this 8 spaces when prior files are 4? Is this a tab rather than spaces? What is the convention in this repo between tabs and spaces?